components/homepage/ProjectCard.vue

```diff
@@ -3,6 +3,7 @@
     <div class="p-8 hover:bg-zinc-100 transition-colors duration-100">
       <!-- Special Tags on the Top Right -->
       <ProjectStatus :project="project" />
+      <ProjectMaturity class="flex justify-end pb-2 px-4" :maturity="project.maturity ? props.project.maturity : 0" />
```

Comment:
maturity is not optional in the schema so it should always be there
```suggestion
      <ProjectMaturity class="flex justify-end pb-2 px-4" :maturity="project.maturity" />
```

Reply:
I thought so too, but it isn't :woman_shrugging: e.g. in the DEDIS file there are several projects without a `maturity` field